### PR TITLE
Update MSW Visual Studio Builds section to reflect currently used ver…

### PR DESCRIPTION
**Pull request by doscott**
There are some whitespace changes here that I had to perform so that the output in Ocular would display correctly. I did the editing using Kate. Please feel free to make or suggest any corrections.

### CODE REVIEW

File: docs/contributing/how-to-release.md [hunk 1]
```diff
@@ -171,20 +172,32 @@ WINDOWS71SDK
...
-Visual Studio 2010  vc100
-Visual Studio 2012  vc110
-Visual Studio 2014  vc120
-Visual Studio 2015  vc140
+    Visual Studio 2008  vc90
+    Visual Studio 2010  vc100
+    Visual Studio 2012  vc110
+    Visual Studio 2014  vc120
+    Visual Studio 2015  vc140
+    Visual Studio 2017  vc150
```
Comment (PBfordev) [inline docs/contributing/how-to-release.md]: According to _buildall.bat_ and _officialbuild.bat_ (and elsewhere), the versions for MSVS 2017 and 2019 should be `vc141` and `vc142`?

File: docs/contributing/how-to-release.md [hunk 2]
```diff
@@ -171,20 +172,32 @@ WINDOWS71SDK
...
+    Visual Studio 2019  vc160
 
 This will build all of the x86 and x64 binaries for the selected compiler version,
 package them in 7z files and calculate the checksums. The 7z files and the
 checksums are output to the build\msw\packages folder.
 
 All of the compiler packages can be built at the same time by executing the
-build\msw\tools\buildall.bat file (including Visual Studio 2000).
+build\msw\tools\buildall.bat file from a command prompt. Each build will be
+launched in it's own shell.
```
Comment (PBfordev) [inline docs/contributing/how-to-release.md]: it's -> its

File: docs/contributing/how-to-release.md [hunk 3]
```diff
@@ -171,20 +172,32 @@ WINDOWS71SDK
 
 If either of these are blank they are set to the default install location.
 
+For Visual Studio 2017 and 2019 the VSxxxxCOMNTOOLS environment variable is not
+set by the installer. Each of these studio version supports multiple toolsets.
```
Comment (PBfordev) [inline docs/contributing/how-to-release.md]: I would change "installer. Each of these studio version supports" to "as these two versions support".

File: docs/contributing/how-to-release.md [hunk 4]
```diff
@@ -171,20 +172,32 @@ WINDOWS71SDK
...
+    Visual Studio 2017  vc150
+    Visual Studio 2019  vc160
 
 This will build all of the x86 and x64 binaries for the selected compiler version,
 package them in 7z files and calculate the checksums. The 7z files and the
 checksums are output to the build\msw\packages folder.
 
 All of the compiler packages can be built at the same time by executing the
-build\msw\tools\buildall.bat file (including Visual Studio 2000).
+build\msw\tools\buildall.bat file from a command prompt. Each build will be
```
Comment (PBfordev) [inline docs/contributing/how-to-release.md]: build\msw\tools -> build\tools\msvs

File: docs/contributing/how-to-release.md (hunk 1 above)
Comment (doscott) [reply docs/contributing/how-to-release.md]: You are correct. I was flipping back and forth between files and picked up the "version" numbers MS uses in the vswhere command. Momentary brain freeze on my part. Will update shortly.

File: docs/contributing/how-to-release.md (hunk 4 above)
Comment (PBfordev) [inline docs/contributing/how-to-release.md]:
```suggestion
 build\tools\msvs\buildall.bat file from a command prompt. Each build will be
```

File: docs/contributing/how-to-release.md (hunk 2 above)
Comment (PBfordev) [inline docs/contributing/how-to-release.md]:
```suggestion
launched in its own shell.
```

File: docs/contributing/how-to-release.md (hunk 1 above)
Comment (PBfordev) [reply docs/contributing/how-to-release.md]: BTW, did you talk with @vadz and has been the final decision made about which binaries for which MSVS versions will be officially released? vadz thought that if 2015-2019 were truly interchangeable, perhaps just one of the three would suffice, see https://github.com/wxWidgets/wxWidgets/pull/1557#issuecomment-532765654

**Comment by vadz** [comment]
Thanks, I'll merge this for now, although there probably will be more changes before the release.

**Pull request closed without merging**